docs/src/main/sphinx/connector/mongodb.rst

```diff
@@ -24,7 +24,7 @@ replacing the properties as appropriate:
 .. code-block:: text
 
     connector.name=mongodb
-    mongodb.connection-url=protocol://user:pass@sample.host:27017/
+    mongodb.connection-url=mongodb://user:pass@sample.host:27017/
```

Comment:
This property accepts `mongodb+srv` too. That is why I approved `protocol`. Can we change to placeholder style? e.g. `{protocol}`

Reply:
That works too.. we just have to maybe also link to the section below then.. since this is an example only